### PR TITLE
 made login auth in frontend

### PR DESCRIPTION
**Pull request by iturnmyscreenon**

### CODE REVIEW

**Comment by rocambille** [comment]
conflicts to resolve

**Comment by iturnmyscreenon** [comment]
conflicts resolved

File: frontend/src/main.jsx [hunk 1]
```diff
@@ -11,14 +12,16 @@ const root = ReactDOM.createRoot(document.getElementById("root"));
...
-      </Routes>
-    </Router>
+    <AuthContextProvider>
+      <Router>
+        <Routes>
+          <Route path="/" element={<App />} />
+          <Route path="/offers/:id" element={<OfferDetails />} />
+          <Route path="/connexion" element={<ConnexionPage />} />
+          <Route path="/createaccount/:type" element={<CreateAccount />} />
+          <Route path="/CandidateArea" element={<CandidateArea />} />
```
Comment (rocambille) [inline frontend/src/main.jsx:22]: you can find a better url :)

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by iturnmyscreenon** [comment]
changed route name

**Pull request merged**